backend/src/main/java/com/ody/mate/service/MateService.java

```diff
@@ -34,15 +34,16 @@ public class MateService {
     private final NotificationService notificationService;
     private final RouteService routeService;
 
+
     @Transactional
-    public MateSaveResponseV2 saveAndSendNotifications(
-            MateSaveRequestV2 mateSaveRequest,
-            Member member,
-            Meeting meeting
-    ) {
+    public MateSaveResponseV2 saveAndSendNotifications(MateSaveRequestV2 mateSaveRequest, Member member, Meeting meeting) {
         if (mateRepository.existsByMeetingIdAndMemberId(meeting.getId(), member.getId())) {
-            throw new OdyBadRequestException("약속에 이미 참여한 회원입니다.");
+            throw new OdyBadRequestException("약속방에 이미 참여한 회원입니다.");
+        }
+        if (meeting.isOverdue()) {
+            throw new OdyBadRequestException("참여 가능한 시간이 지난 약속방에 참여할 수 없습니다.");
```

Comment:
[제안]
`약속`으로 표현 통일해주면 더 좋을 것 같아요!
```suggestion
            throw new OdyBadRequestException("약속에 이미 참여한 회원입니다.");
        }
        if (meeting.isOverdue()) {
            throw new OdyBadRequestException("참여 가능한 시간이 지난 약속에 참여할 수 없습니다.");
```

Reply:
좋은 것 같아요 👍 
"약속방"이라는 단어를 사용하는 몇가지 파일들이 존재해 일괄로 "약속"이란 단어를 사용하도록 변경했습니다 !